### PR TITLE
Updates to navigation and content.

### PR DESCRIPTION
**Pull request by justinegeffen**
- Updates to the navigation for Installation guides.
- Deprecation notice added to Ubuntu 16.04 and RHEL 6. 

Fixes: https://mattermost.atlassian.net/browse/MM-26651.

### CODE REVIEW

**Comment by justinegeffen** [comment]
@svelle, you can view the proposed changes at https://mattermost.github.io/docs/justinegeffen/index.html.

**Comment by svelle** [comment]
@justinegeffen this looks good. One thing that I noticed is that the 16.04 and rhel 6 guides are still linked here: https://mattermost.github.io/docs/justinegeffen/install/ee-install.html

might be good removing them from there as well. what do you think?

File: source/guides/administrator.rst [hunk 1]
```diff
@@ -27,31 +25,19 @@ Learn how to get Mattermost running on your environment.
    :glob:
 
    /install/requirements.rst
-   Installing on Ubuntu 16.04 LTS </install/install-ubuntu-1604.rst>
-   Installing on Ubuntu 18.04 LTS </install/install-ubuntu-1804.rst>
+   /install/ee-install*
+   /install/desktop*
    Installing on Kubernetes </install/install-kubernetes.rst>
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply source/guides/administrator.rst]: I've reordered the list.

File: source/guides/administrator.rst [hunk 2]
```diff
@@ -27,31 +25,19 @@ Learn how to get Mattermost running on your environment.
    :glob:
 
    /install/requirements.rst
-   Installing on Ubuntu 16.04 LTS </install/install-ubuntu-1604.rst>
-   Installing on Ubuntu 18.04 LTS </install/install-ubuntu-1804.rst>
+   /install/ee-install*
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply source/guides/administrator.rst]: Agreed; I've moved it.

**Comment by justinegeffen** [comment]
@jasonblais, thanks for all the feedback. I've made some changes and wanted to raise the following:
1. There is a pending PR for Ubuntu 20.04 which is still in review. It doesn't block this PR.
2. There is another update pending for the installation guide. Would you consider this PR good to merge for this iteration?

File: source/guides/administrator.rst [hunk 3]
```diff
@@ -27,32 +27,22 @@ Learn how to get Mattermost running on your environment.
...
    Installing on RHEL 7 </install/install-rhel-7.rst>
    Installing on CentOS, Oracle Linux, and Scientific Linux </install/install-centos-oracle-scientific.rst>
    Instructions On Setting Up a Socket-based Mattermost Database </install/sockets-db.rst>
    /install/ee-install*
    /install/transport-encryption/config.rst
    /install/deploy-bitnami*
    /install/docker-local*
-   /install/prod-docker*
    /install/docker-ebs*
    /install/deploy-cloudron*
```
Comment (jasonblais) [inline source/guides/administrator.rst:41]: Do we intend to remove it from the install section as it was added to community-managed section?

File: source/guides/administrator.rst [hunk 4]
```diff
@@ -27,32 +27,22 @@ Learn how to get Mattermost running on your environment.
...
    Installing Mattermost Team Edition in GitLab Helm Chart </install/install-mmte-helm-gitlab-helm.rst>
-   /install/trouble_mysql.rst
    /install/desktop*
    /install/desktop-msi-gpo*
-   /install/desktop-managed-resources*
```
Comment (jasonblais) [inline source/guides/administrator.rst:49]: Intentional move?

File: source/install/install-rhel-6.rst [hunk 5]
```diff
@@ -4,7 +4,11 @@
 Installing Mattermost on RHEL 6
 =================================
 
-You can also use these instructions to install Mattermost on CentOS 6, Oracle Linux 6, or Scientific Linux 6. With the exception of the operating system that you install, the process is identical.
+.. warning::
+  RHEL 6 is approaching its End of Life in December 2020.
+  Because of this, we don't recommend installing new instances of Mattermost on RHEL 6. Rather use a newer version such as RHEL 8.
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply source/install/install-rhel-6.rst:9]: Not yet, no. There is an open issue for it though.

File: source/guides/administrator.rst (hunk 3 above)
Comment (justinegeffen) [reply source/guides/administrator.rst:41]: Yep - the doc itself specifies that it's a community-managed document so it seemed like a better fit.

File: source/guides/administrator.rst (hunk 4 above)
Comment (justinegeffen) [reply source/guides/administrator.rst:49]: Nope - thanks for catching it!

File: source/guides/administrator.rst (hunk 3 above)
Comment (jasonblais) [reply source/guides/administrator.rst:41]: :+1: Let's remove from the installation navigation section

File: source/install/install-mmte-helm-gitlab-helm.rst [hunk 6]
```diff
@@ -5,23 +5,24 @@ How to install Mattermost Team Edition Helm Chart in a GitLab Helm Chart deploym
...
 Prerequisites
-----------------------------
+--------------
 
   - A running Kubernetes cluster.
   - `Helm v2 <https://helm.sh/docs/intro/install/)>`_
-  - `Tiller <https://rancher.com/docs/rancher/v2.x/en/installation/ha/helm-init/>`_ (the Helm server-side component)     installed on the cluster.
+  - `Tiller <https://rancher.com/docs/rancher/v2.x/en/installation/ha/helm-init/>`_ (the Helm server-side component) installed on the cluster.
+
+.. note::
```
Comment (amyblais) [inline source/install/install-mmte-helm-gitlab-helm.rst:18]: The note label is not visible on preview.

File: source/guides/administrator.rst [hunk 7]
```diff
@@ -27,32 +27,22 @@ Learn how to get Mattermost running on your environment.
...
    Instructions On Setting Up a Socket-based Mattermost Database </install/sockets-db.rst>
    /install/ee-install*
    /install/transport-encryption/config.rst
    /install/deploy-bitnami*
    /install/docker-local*
-   /install/prod-docker*
    /install/docker-ebs*
    /install/deploy-cloudron*
    Installing Mattermost Team Edition in GitLab Helm Chart </install/install-mmte-helm-gitlab-helm.rst>
-   /install/trouble_mysql.rst
```
Comment (amyblais) [inline source/guides/administrator.rst:46]: Do we want to remove this?

Comment (justinegeffen) [reply source/guides/administrator.rst:46]: I've moved it into the Troubleshooting section. The style is different (more content, fewer FAQ-style entries) but it seemed like a better fit in terms of subject matter. What are your thoughts?

File: source/install/install-rhel-6.rst (hunk 5 above)
Comment (justinegeffen) [inline source/install/install-rhel-6.rst:9]:
```suggestion
  Because of this, we don't recommend installing new instances of Mattermost on RHEL 6.
```

Comment (justinegeffen) [reply source/install/install-rhel-6.rst:9]: Removed recommendation for RHEL 8 until we have a guide.

**Pull request merged**